app/js/actions/addDrugOrder.js

```diff
@@ -1,15 +1,16 @@
-import axios from 'axios';
 import {
   POST_DRUG_ORDER_LOADING,
   POST_DRUG_ORDER_SUCCESS,
   POST_DRUG_ORDER_FAILURE,
 } from './actionTypes';
+import http from '../config';
 import networkError from './networkError';
 import loading from './loading';
 
-const contextPath = window.location.href.split('/')[3];
-const apiBaseUrl = `/${contextPath}/ws/rest/v1`;
-
+export const postDrugOrderLoading = status => ({
+  type: POST_DRUG_ORDER_LOADING,
+  status,
+});
```

Comment:
I guess lines 10 -13 are no more needed

Reply:
This is being worked on in another ticket [here](https://issues.openmrs.org/browse/OEUI-84) by @fuchodeveloper  and is not part of this task

Reply:
Alright

Reply:
Currently working on this

Reply:
Okay